lesson_02/script.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+const li = document.createElement("li");
+
+const a = document.createElement("a");
+a.text = "これです";
+a.href = "1.html";
+
+const img = document.createElement("img");
+img.alt = "ブックマーク";
+
+const ul = document.getElementById("target");
```

Comment:
The code is
For each variable definition
For each instruction to the element
For each DOM operation
I think it will be easier to read if you write it separately.